interfaces/builtin/shared_memory_test.go

```diff
@@ -71,16 +83,29 @@ slots:
   shared-memory: foo
   write: [ bar ]
   read: [ bar-ro ]
+  private: false
  shmem-wildcard:
   interface: shared-memory
   shared-memory: foo-wildcard
   write: [ bar* ]
   read: [ bar-ro* ]
+  private: false
```

Comment:
both here and in the consumer yaml, I would add `private: false` only half of the time, so that we verify that the default value is correctly assumed.

Reply:
These tests specify all attributes on the plugs/slots because they are not running `BeforePreparePlug`/`BeforePrepareSlot`. We have other tests that test that the `BeforePrepare*` methods set defaults appropriately. These tests are effectively assuming a sanitised set of attributes.

Reply:
Oh, right! Ignore me then :-)